tutorminio/patches/openedx-lms-common-settings

```diff
@@ -0,0 +1,9 @@
+# LMS-specific media storage
+PROFILE_IMAGE_BACKEND = {
+    "class": "{{ DEFAULT_FILE_STORAGE }}",
+    "options": {
+        "bucket_name": "{{ MINIO_BUCKET_NAME }}",
+        "querystring_auth": False,
```

Comment:
why should this be False?

Reply:
[these](https://github.com/overhangio/tutor-minio/issues/46#:~:text=As%20I%20understood%20from%20the%20shallow%20investigation%20%2D%20querystring_auth%20doesn%27t%20work%20because%20_get_profile_image_urls%20appends%20additional%20querystring%20kwarg%20%26v%3D....%20If%20you%20remove%20it%20from%20the%20URL%20%2D%20signature%20works%20as%20expected.) were the reporter's findings. We still need to test it on our end.

Reply:
Right, I think that should be verified before making this PR ready for review.